examples/docking-protein-peptide/docking-protein-peptide.cfg

```diff
@@ -55,7 +55,7 @@ end_fle_2_1 = "11"
 ssdihed = "alphabeta"
 
 [caprieval]
-reference = 'data/1nx1_refe.pdb'
```

Comment:
There is one more reference instance on line 41 in that file that needs to be corrected

Reply:
And while at it, please add the reference to the protein-ligand cfg last caprieval module

```
[caprieval]
reference_fname = 'data/target.pdb'
```